tests/test_other.py

```diff
@@ -288,23 +288,54 @@ def test_empty_column_bad(basic, basic_good_with_empty):
 
 
 @pytest.fixture()
```

Comment:
Tried to follow your lead with the `test_complex_good`... I extended it to include specific tests for the kinda weird 'conditional' cell rules (`NotNullIfCompare`, `OtherCantExist`, and `OtherMustExist`)... I then attempted to replicate the approach to ensure that a bad complex file will trip when expected... I feel the way I extended the test makes them a bit hard to parse since it's trying to check several things at once, but opted to stay the course since we are trying to test the combination of these things aferall!